pkg/test/framework/resource/flags.go

```diff
@@ -202,9 +208,9 @@ func init() {
 			"Secret should already exist when used with istio.test.stableNamespaces.")
 	flag.Uint64Var(&settingsFromCommandLine.MaxDumps, "istio.test.maxDumps", settingsFromCommandLine.MaxDumps,
 		"Maximum number of full test dumps that are allowed to occur within a test suite.")
-	flag.BoolVar(&settingsFromCommandLine.EnableDualStack, "istio.test.enableDualStack", settingsFromCommandLine.EnableDualStack,
-		"Deploy Istio with Dual Stack enabled.")
 	flag.StringVar(&settingsFromCommandLine.HelmRepo, "istio.test.helmRepo", settingsFromCommandLine.HelmRepo, "Helm repo to use to pull the charts.")
+	flag.Var(&settingsFromCommandLine.IPFamilies, "istio.test.IPFamilies",
+		"IP families (IPv6, IPv4) to test with. If both specified, dualstack config will be used. The order the families are defined indicates precedence.")
```

Comment:
If a user accidentally passes a wrong value (or repeated IPFamily) I'm not sure how easy the resulting error would be to figure out that input was wrong. So, shall we add a validation for the input?

Reply:
I have added some minimal validation, however for other cases the services would fail to deploy and the logs clearly show that